docs/integrations/decoration-protocol.md

```diff
@@ -109,11 +109,20 @@ export interface PublishDecorationsParams {
    * Use empty list to clear all decorations.
    */
   options: DecorationOptions[];
+
+  /**
+   * Used to distinguish if inline decorations from the
+   * line ones as otherwise they will get replaced. It's
+   * not sent if client doesn't support inline decorations.
```

Comment:
Final nit, maybe make it super clear here what the behavior is. Meaning these will never be sent in general since a client that doesn't support inline decorations will never be sent these decorations? or they will be sent these decorations, but the `inline` value won't be there? It's a bit ambiguous right now if you don't have the full context. Probably best to _over_ communicate here ha.

Reply:
**Perfect**

Reply:
:rocket: